docs/book/src/tooling/clusterctl.md

```diff
@@ -1 +1,69 @@
-# Clusterctl
+# Using `clusterctl` to create a cluster from scratch
```

Comment:
Above this line, we should add the following deprecation notice:

```html
<aside class="note warning">

<h1>Deprecation Notice</h1>

This utility has been deprecated in v1alpha2 and it'll be removed in a future version in favor. 

</aside>
```

Reply:
done.